homeassistant/components/light/isy994.py

```diff
@@ -72,7 +72,9 @@ def turn_on(self, brightness=None, **kwargs) -> None:
     @property
     def state_attributes(self):
```

Comment:
I'll take a look at this now. Some history on this, the ISY component was written right around the same time that the Entity abstract classes were being cleaned up. That is why somethings are overwritten that probably shouldn't be. In fact, I don't believe that device_state_attributes existed at the time that the ISY component was originally written.